backend/oasst_backend/tree_manager.py

```diff
@@ -924,14 +929,44 @@ def _insert_default_state(
             active=True,
         )
 
+    def export_trees_to_file(
+        self,
+        message_tree_ids: list[str],
+        file=None,
+        reviewed: bool = True,
+        deleted: bool = False,
+        use_compression: bool = True,
+    ) -> None:
+        for message_tree_id in message_tree_ids:
+            messages: List[Message] = pr.fetch_message_tree(message_tree_id, reviewed, deleted)
+            tree: tree_export.ExportMessageTree = tree_export.build_export_tree(message_tree_id, messages)
+            if file:
+                tree_export.write_tree_to_file(file, tree, use_compression)
+            else:
+                logger.info(json.dumps(jsonable_encoder(tree), indent=2))
```

Comment:
not specifying a file should not write the message via logger ... I had `sys.stdout` in mind when I wrote in case no file is specified for the cli app.